### DISCUSSION TITLE
Timeout

### DISCUSSION THREAD

**Discussion by StanislavJochman** [filebrowser/filebrowser#2284] [Q&A] [answered]
Hi is there a way how to change timeout of logout? It is a bit short for my needs and sometimes when I watch long video it times out.

Thanks for answer

Best regards
Stanislav Jochman

**Comment by gmc-72** [comment]
I'm needing this too.  Uploading a 65GB single file from my laptop to the server ... it should take about 4 hours, but getting logged out before it completes.

Worse, it shows an incomplete file with no indication it's a failed upload.  I've already deleted several files that appeared to have uploaded, but which had actually timed out near to completion - rendering them useless and the data lost : (

Timeout is getting in the way of the Apps core function!

 - I can't see any setting to adjust/disable the timeout
 - I can't see any setting to auto-delete failed uploads
 - Timeout should NEVER be allowed to interrupt a transfer in progress
 
 Please someone advise or fix : (

**Comment by ramiresviana** [comment, marked as answer]
The session timeout can now be changed by running File Browser with the flag `--token-expiration-time=2H`.
